config/routes.rb

```diff
@@ -70,9 +70,11 @@
 
   namespace :bulk_update, path: "bulk-update" do
     resources :recommendations_uploads, only: %i[new create edit update], path: "recommend", path_names: { new: "upload", edit: "upload-changes" } do
-      get "upload-summary", to: "recommendations_uploads#show", as: "summary"
       get "check-pending-updates", to: "recommendations_uploads#check", as: "check"
       get "confirmation"
+      get "download_errors"
+      get "fix-errors"
```

Comment:
cc @johndoates - the URL in the prototype is `fix-errors`, but the button and page heading both say `Review errors`. Just wondering whether `review-errors` in the URL would be more consistent. What do you think?

Reply:
Also, `fix-errors` feels weird for a `get` route as it won't actually change anything 🤷